specification/storagesync/resource-manager/readme.typescript.md

```diff
@@ -6,7 +6,7 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 ``` yaml $(typescript)
 typescript:
   azure-arm: true
-  package-name: "@azure/arm-storagesync"
```

Comment:
Please revert these changes. These relate to the TypeScript SDKs.

Reply:
No. These should remain as they are. Let me look around this folder and make sure you have everything configured correctly.

Reply:
Everything looks fine. Just revert changes to this file. We're going to be publishing these TypeScript-based SDKs this week.

Reply:
You can see the generated SDK here: https://github.com/Azure/azure-sdk-for-js/tree/master/packages/%40azure/arm-storagesync.